### PR TITLE
Feature/users endpoint

### PR DESCRIPTION
**Pull request by chubchenko**

### CODE REVIEW

File: app/services/vertigo/rtm/users_search_query.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+      private
+
+      attr_reader :relation, :params
+
+      def users_by_prefix
+        query = params['q'].to_s.strip
+
+        if query.length >= MIN_QUERY_LENGTH
+          @relation.where("#{Vertigo::Rtm.user_name_column} LIKE :query", query: "%#{query}")
```
Comment (RomanMukhin) [inline app/services/vertigo/rtm/users_search_query.rb]: What about % at the end of query string, do we match only the end of name field?

Comment (RomanMukhin) [inline app/services/vertigo/rtm/users_search_query.rb]: I would use something like lower(#{Vertigo::Rtm.user_name_column}) and lowercase the query string.

**Comment by RomanMukhin** [comment]
👍

**Comment by vshmyhlo** [comment]
:+1:

**Pull request merged**